consul_config.py.ctmpl

```diff
@@ -171,3 +171,8 @@ WEBSOCKETS_SERVER_URL = '''{{template "KEY" "websockets_server_url"}}'''
 SESSION_REMEMBER_ME_DURATION = 365
 
 LISTEN_DUMP_TEMP_DIR_ROOT = '''{{template "KEY" "listen_dump_temp_dir"}}'''
+
+# Feature flag: only enable follow button in beta and test
+{{if not eq (env "DEPLOY_ENV") "prod"}}
+FEATURE_FOLLOW_BUTTON = True
+{{end}}
```

Comment:
Your thoughts on this @paramsingh ?
I haven't been able to test it.

Do you have a way to test this locally?

 Maybe we can try on test.lb first?
We might need to do the db migration on the test db if so.